### PR TITLE
WatchfaceAnalog: add ble icon

### PR DESCRIPTION
**Pull request by Itai-Nelken**
The only difference between this pr and #1331 is that the ble icon is aligned differently.
In this pr it is aligned next to the battery icon, and in #1331 it is aligned in the bottom left corner.
InfiniSim_2022-11-12_125028

Personally I prefer it this way (next to the battery icon), but I'm obviously biased :).

### CODE REVIEW

**Review by Riksu9000 — APPROVED** [review]
Fixes #424

**Comment by Itai-Nelken** [comment]
Also closes #1331.

**Comment by Itai-Nelken** [comment]
Is there anything I have to do other than to wait until a maintainer has time and keep the pr updated?

File: src/displayapp/screens/WatchFaceAnalog.cpp
Comment (Riksu9000) [inline src/displayapp/screens/WatchFaceAnalog.cpp:77]: Aligning this to `OUT_LEFT_TOP` of the batteryIcon with offset 5 I think would be the same position, but a bit more correct.

**Review by Riksu9000 — COMMENTED** [review]
Mostly waiting for another approval. I may merge this soon anyway.

**Review by FintasticMan — APPROVED** [review]
LGTM

**Review by JF002 — APPROVED** [review]

File: src/displayapp/screens/WatchFaceAnalog.cpp
Comment (Itai-Nelken) [reply src/displayapp/screens/WatchFaceAnalog.cpp]: I did that (the offset to make it look the same was -18), but I accidentally committed it along with the merge of develop into my fork. Is that ok or should I somehow squash all the commits in this pr into a single one, and if so how do I do it?

Comment (Riksu9000) [reply src/displayapp/screens/WatchFaceAnalog.cpp]: https://github.com/InfiniTimeOrg/InfiniTime/blob/develop/CONTRIBUTING.md#commit-conventions Clean merge commits are fine, they'll be automatically squashed, however conflicts should be resolved by rebasing. When on this branch you run `git rebase -i develop`, where develop is a branch that has the latest changes from develop in this repo. The exact branch name may depend on your local setup. I suggest reverting this change and we'll merge it as it was previously.

Comment (Itai-Nelken) [reply src/displayapp/screens/WatchFaceAnalog.cpp:77]: I reverted the commit using the `OUT_LEFT_TOP`.

Comment (Riksu9000) [reply src/displayapp/screens/WatchFaceAnalog.cpp:77]: The last commit is a mistake. You should rebase again and drop the commit.

Comment (Itai-Nelken) [reply src/displayapp/screens/WatchFaceAnalog.cpp:77]: done.

**Pull request merged**
